scheduling/allocator.rb

```diff
@@ -24,7 +13,8 @@ def self.allocate(vm, storage_volumes, distinct_storage_devices: false, gpu_coun
       gpu_count,
       gpu_device,
       vm.ip4_enabled,
-      target_host_utilization,
+      Config.allocator_target_host_utilization,
```

Comment:
what about we don't add `target_premium_host_utilization` but set the target_host_utilization to either `Config.allocator_target_host_utilization` or `Config.allocator_target_premium_host_utilization`.

Reply:
If we set the single value to %85, it means that “premium VMs can be allocated on both standard/premium hosts up to 85%,” not that “premium hosts can be filled up to 85%”